src/Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IsPackable>false</IsPackable>
+		<NoWarn>$(NoWarn);MSML_ParameterLocalVarName;SA1028</NoWarn>
```

Comment:
SA1028 is about code comments not formatting correctly. That seems like something we should be able to fix (either here or in a follow up PR)

Reply:
Yup. My plan is to do it in a follow up PR. SA1028 and a bunch of the other warnings are easily fixable